ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/PreparedAuditOperation.java

```diff
@@ -88,7 +94,28 @@ private String preparedWithValues()
         Queue<ByteBuffer> values = new LinkedList<>(options.getValues());
         for (ColumnSpecification column : options.getColumnSpecifications())
         {
-            String value = column.type.asCQL3Type().toCQLLiteral(values.remove(), Server.CURRENT_VERSION);
+            ByteBuffer serializedValue = values.remove();
+            String value = null;
+            if (column.type instanceof UTF8Type || column.type instanceof AsciiType)
+            {
+                value = "'" + column.type.getString(serializedValue) + "'";
+            }
+            else
+            {
+                if (serializedValue.hasRemaining())
+                {
+                    if (column.type instanceof TimestampType)
+                    {
+                        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.getDefault(Locale.Category.FORMAT));
+                        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+                        value = sdf.format(TimestampSerializer.instance.deserialize(serializedValue));
+                    }
+                    else
+                    {
+                        value = column.type.getString(serializedValue);
+                    }
+                }
+            }
```

Comment:
This block should be refactored and preferably placed in a separate class.

Reply:
Added ColumnDataPrinter util class...
While testing that a new test dependency was added - JUnitParams, which enables data providers to tests... This is not used in the c3+ code for now, but maybe that can be useful for other tests in the future?